3.6/alpine/docker-entrypoint.sh

```diff
@@ -253,17 +253,27 @@ if [ "$1" = 'rabbitmq-server' ] && [ "$shouldWriteConfig" ]; then
 	)
 
 	# determine whether to set "vm_memory_high_watermark" (based on cgroups)
-	if [ -r /sys/fs/cgroup/memory/memory.limit_in_bytes ] && [ -r /proc/meminfo ]; then
-		memLimitB="$(< /sys/fs/cgroup/memory/memory.limit_in_bytes)"
-		memLimitKb="$(( memLimitB / 1024 ))"
-
+	memTotalKb=
+	if [ -r /proc/meminfo ]; then
 		memTotalKb="$(awk -F ':? +' '$1 == "MemTotal" { print $2; exit }' /proc/meminfo)"
-
-		if [ "$memLimitKb" -gt "$memTotalKb" ]; then
+	fi
+	memLimitB=
+	memLimitKb=
+	if [ -r /sys/fs/cgroup/memory/memory.limit_in_bytes ]; then
+		memLimitB="$(< /sys/fs/cgroup/memory/memory.limit_in_bytes)"
+		if [ "$memLimitB" = '0' ]; then
+			# it appears that "WARNING: No kernel memory limit support" leads to "/sys/fs/cgroup/memory/memory.limit_in_bytes" containing the value "0"
```

Comment:
Ok, this is confirmed to be FALSE; time to keep digging (although this PR's changes are probably still sane, possibly minus the `0` special-casing):
```console
$ docker info > /dev/null
WARNING: No memory limit support
WARNING: No swap limit support
WARNING: No kernel memory limit support
WARNING: No oom kill disable support
WARNING: No cpu cfs quota support
WARNING: No cpu cfs period support
$ docker run --rm bash ls -l /sys/fs/cgroup/memory
ls: /sys/fs/cgroup/memory: No such file or directory
```